griptape/drivers/prompt/amazon_bedrock_prompt_driver.py

```diff
@@ -98,8 +105,11 @@ def _base_params(self, prompt_stack: PromptStack) -> dict:
             "modelId": self.model,
             "messages": messages,
             "system": system_messages,
-            "inferenceConfig": {"temperature": self.temperature},
-            "additionalModelRequestFields": self.additional_model_request_fields,
+            "inferenceConfig": {
+                **({"temperature": self.temperature} if self.temperature is not None else {}),
+                **({"topP": self.top_p} if self.top_p is not None else {}),
+            },
+            "additionalModelRequestFields": self.additional_params,
```

Comment:
This seems awkward. Would anyone ever need to pass additional params to `inferenceConfig`? (Like wouldn't top k go there?)

Reply:
Ah good point -- maybe we can take inspiration from how Bedrock has split out their fields here? 
`inferenceConfig` = top level Driver fields -- common to all models.
`additionalmodelRequestFields` = `additional/native_params` -- model specific parameters.